migrations/migrator.go

```diff
@@ -151,6 +151,40 @@ func (m *Migrator) Up(ctx context.Context) error {
 	}
 }
 
+func (m *Migrator) Until(ctx context.Context, version int) error {
+
+	if err := m.initSchema(ctx); err != nil {
```

Comment:
These steps are already performed by upByOne function.
Can you use results of this function instead of rewrite the code?